be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/ParamsHandler.scala

```diff
@@ -54,14 +54,14 @@ object ParamsHandler {
     case Left(jsonRpcMessage: Subscribe) =>
       // ActorFlow.ask(clientActorRef)(makeMessage = (el, replyTo: ActorRef) => SubscribeTo(channel))
       clientActorRef ! ClientActor.SubscribeTo(jsonRpcMessage.channel)
-      Right(PipelineError(-100, "FIXME: should use akka ask pattern to create an ActorFlow"))
+      Right(PipelineError(-100, "", Some(-111))) // FIXME: should use akka ask pattern to create an ActorFlow
```

Comment:
what's going on here? I don't remember how this should be fixed 🤔

Reply:
Was waiting for akka ask pattern to work in the system. I resolved the issue before even seeing your message ^^. See #441